.github/workflows/protocol-sim.yml

```diff
@@ -2,11 +2,14 @@ name: Protocol Sims
 # Sims workflow runs multiple types of simulations (nondeterminism, import-export, after-import, multi-seed-short)
 # This workflow will run on all pushes to main, if a .go, .mod or .sum file have been changed
 on:  # yamllint disable-line rule:truthy
+  pull_request:
+    paths:
+      - 'protocol/**'
   push:
     branches:
       - main
-      - 'release/[a-z]+/v[0-9]+.[0-9]+.x'  # e.g. release/ibctestnet/v1.0.x
-      - 'release/v[0-9]+.[0-9]+.x'  # e.g. release/v0.0.x
+      - 'release/protocol/v0.[0-9]+.x'  # e.g. release/protocol/v0.1.x
```

Comment:
This doesn't prevent the scenario described [here](https://dydx-team.slack.com/archives/GTGEVSCUU/p1696523249079489) right? 

> Protocol Sims are run on indexer only commits ([e.g.](https://github.com/dydxprotocol/v4-chain/actions/runs/6357582054)), which takes ~40min

Can we do something like 


```
  push:
    paths: 
      - 'protocol/**'
    branches:
      - main
      - 'release/protocol/v0.[0-9]+.x'  # e.g. release/protocol/v0.1.x
```

Is it an "or" or "and" logic between `paths` and `branches` here?

Reply:
Ah oops, nice catch.